tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -191,6 +191,9 @@ cc_library(
         "//tensorflow:linux_ppc64le": [
             "@llvm-project//llvm:PowerPCCodeGen",  # fixdeps: keep
         ],
+        "//tensorflow:linux_s390x": [
```

Comment:
I'm not sure, I only verify that it could successfully build on s390x machine (and has no effect on x86_64), JIT compile could work and test cases including `//tensorflow/python/compiler/xla:jit_test` will pass. But there still a number of failures in `//tensorflow/compiler/` unit test module. Do you need me to do any other verification?